src/streetscapes/sources/downloader.py

```diff
@@ -0,0 +1,111 @@
+import hashlib
+import shutil
+from pathlib import Path
+import datetime
+from rich.progress import track
+
+
+class ImageDownloader:
```

Comment:
This seems outdated. CLI for downloading images is added in https://github.com/Urban-M4/streetscapes/pull/80

I guess we should clean this up (is on the TODO list). Agreed?